exercises/sum.cpp

```diff
@@ -6,3 +6,16 @@
   Insert the second number: 2
   Sum: 3
 */
+#include <iostream>
+using namespace std;
+int main(){
+  int num1,num2,sum;
+  num1=num2=0;
+  cout<<"Insert the first number: ";
+  cin>>num1;
+  cout<<"Insert the second number: ";
+  cin>>num2;
+  sum=num1+num2;
+  cout<<"Sum:"<<sum<<"\n";
```

Comment:
la variabile "sum" è inutile, puoi benissimo scrivere `cout << "Sum:" << num1+num2 << ecc.`
metti degli spazi nel codice e muovi questo codice in una funzione esterna chiamata sum()